BloodDonor/Modules/DonationList/DonationListView.swift

```diff
@@ -10,10 +10,13 @@ import SwiftUI
 struct DonationListView: View {
     var body: some View {
         NavigationView {
-            ZStack {
-                Color.blue
+            VStack {
+                Text("👀 Brak donacji")
+                    .font(.largeTitle.bold())
+                Text("Trochę tu pusto, pora dodać jakąs donacje!")
+                    .font(.callout)
             }
-            .navigationTitle("Donations")
+            .navigationTitle("Lista donacji")
```

Comment:
This should be changed to the agreed name of 'Dziennik donacji'

Reply:
oke will do